modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/request/SearchResponseImpl.java

```diff
@@ -75,8 +72,14 @@ public SearchSettings getSearchSettings() {
 		return _searchSettings;
 	}
 
-	public Optional<String> getSpellCheckSuggestionOptional() {
-		return SearchStringUtil.maybe(_hits.getCollatedSpellCheckResult());
+	public String getSpellCheckSuggestion() {
+		String collatedSpellCheckResult = _hits.getCollatedSpellCheckResult();
```

Comment:
@julschong `SearchStringUtil.maybe` performs `StringUtil.trim()` on the string first. When you modify it, you should keep this.

By the way, since `SearchStringUtil.maybe` is used in a lot of places in `portal-search-web`, I recommend that you don't directly remove this usage. Instead, keep the `Optional` usage in the method body, and modify it when you work on `portal-search-web`.